packages/online-editor/src/envelopeLocator/EditorEnvelopeLocatorApi.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2023 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export type EditorEnvelopeConfig = {
+  type: string;
+  filePathGlob: string;
+  resourcesPathPrefix: string;
+  path: string;
+  title: string;
+  description: string;
+};
```

Comment:
```suggestion
export type EditorEnvelopeConfig = {
  extension: string;
  filePathGlob: string;
  editor: {
    resourcesPathPrefix: string;
    path: string;
  },
  card: {
    title: string;
    description: string;
  }
};
```

Please change this type to the structure above. It's easier to group related properties inside a nested object. Also change the necessary places after this change.